### DISCUSSION TITLE
How to load bundles into the application in runtime after build?

### DISCUSSION THREAD

**Discussion by jon9090** [webpack/webpack#16212] [Q&A]
I want to load two js bundles generated by webpack (`app1.js`, `app2.js`) into my application (`host.js`) context. (I mean after the build by giving a string path of the javascript bundle file)

Something like this possible?
```
const apps = ['path/to/app1.js', 'path/to/app2.js'];
import(apps[0]).then(m => {
  m.fn(...);
});
```

**Comment by alexander-akait** [comment]
If you need module federation, please read https://webpack.js.org/concepts/module-federation/, anyway you can it using `__webpack_load_chunk__`, for example:
```
__webpack_load_chunk__("chunk-containing-module.js").then(() => __webpack_require__("./module"))
```
